samtranslator/schema/aws_serverless_api.py

```diff
@@ -1,80 +1,98 @@
+from __future__ import annotations
+
 from typing import Optional, Any, Dict, Union, List
 
 from typing_extensions import Literal
 
-from samtranslator.schema.common import PassThrough, BaseModel, SamIntrinsicable
+from samtranslator.schema.common import PassThrough, BaseModel, SamIntrinsicable, get_prop
+
+resourcepolicy = get_prop("sam-property-api-resourcepolicystatement")
```

Comment:
Why all small letters? kinda hard to read

Reply:
"Reasons"

It was the low-thinking approach for doing this that I chose in the beginning, so I'm sticking with it. The names match exactly the model names, except in lowercase. It's super tedious to do so went with whatever is easy to write. Could've been `_<model>` or whatever.